semantic error: The `async` keyword is currently not supported
 --> tests/errors/test_async.py:1:1 - 2:17
  |
1 |    async def test_async():
  |    ^^^^^^^^^^^^^^^^^^^^^^^...
...
  |
2 |        print("done")
  | ...^^^^^^^^^^^^^^^^^ 
